tests/python/README.md

```diff
@@ -88,7 +88,7 @@ To backup DB and data volume, please use commands below.
 
 ```console
 docker exec test_cvat_server_1 python manage.py dumpdata --indent 2 --natural-foreign --exclude=auth.permission --exclude=contenttypes --exclude=django_rq > shared/assets/cvat_db/data.json
-docker exec test_cvat_server_1 tar -cjv /home/django/data > shared/assets/cvat_db/cvat_data.tar.bz2
+docker exec test_cvat_server_1 tar --exclude "/home/django/data/cache" --exclude "events.db" --exclude "logstash.db" -cjv /home/django/data > shared/assets/cvat_db/cvat_data.tar.bz2
```

Comment:
I think the latter two excludes shouldn't be necessary post-#9015, no?

Reply:
Yeah, seems to be true.